tests/util/xcm.ts

```diff
@@ -112,3 +113,37 @@ export async function registerForeignAsset(
     registeredAsset,
   };
 }
+
+export function descendOriginFromAllOnes(context: DevTestContext) {
```

Comment:
Shall we just create a function descendOriginFromAddress?

Reply:
Yeah that's probably better